box-webhook-aws-sample/box-webhook-aws-sns/src/main/java/com/box/samples/aws/sns/BoxAWSSNSLambda.java

```diff
@@ -0,0 +1,320 @@
+package com.box.samples.aws.sns;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import com.amazonaws.services.sns.AmazonSNS;
+import com.box.samples.aws.sns.logging.LambdaLoggerSlf4j;
+import com.box.samples.aws.sns.utils.FileUtils;
+import com.box.samples.aws.sns.utils.StringUtils;
+import com.box.samples.aws.sns.utils.ValidationUtils;
+import com.box.samples.aws.sns.validation.Validation;
+import com.box.samples.aws.sns.validation.ValidationException;
+import com.box.sdk.BoxAPIConnection;
+import com.box.sdk.BoxAPIException;
+import com.box.sdk.BoxFile;
+import com.box.sdk.BoxFile.Info;
+
+/**
+ * A AWS Lambda implementation.
+ *
+ * @author Stanislav Dvorscak
```

Comment:
please remove all author tags.  People should go to box forums for support rather contacting you.